src/client/pages/ResetPasswordSessionExpiredPage.tsx

```diff
@@ -32,7 +32,7 @@ export const ResetPasswordSessionExpiredPage = () => {
       formPageTrackingName="reset-password-session-expired"
     >
       <MainBodyText>
-        The link we sent you was valid for 30 minutes and it has now expired.
+        The link we sent you was valid for 60 minutes and it has now expired.
```

Comment:
a lot (maybe all) of these `*SessionExpiredPage` pages don't seem to be being used currently. I could test my change by going to the url manually but when the token actually is invalid it just stays in the regular page (password reset in this case) and display a different "session expired" message